src/components/core/DpDataTable/DpDataTable.vue

```diff
@@ -253,6 +459,12 @@ export default {
   },
 
   methods: {
+    addHoveredClass(idx) {
+      const tableRow = this.$refs.tableRows[idx]
+
+      return tableRow.$el.classList.add('is-hovered-content')
+    },
```

Comment:
I wonder why it's done like that. May I please You to try if it works the same if You just remove the hover-Events (skip adding the hover-class). In the CSS it looks like it should work.

Unfortunately I can't test it by myself at the moment because my Local branch is messed up :-/

Reply:
:facepalm: I see. Sorry for bothering You with that